response_operations_ui/views/reporting_units.py

```diff
@@ -225,6 +227,18 @@ def confirm_change_enrolment_status(ru_ref):
                            change_flag=request.args['change_flag'])
 
 
+@reporting_unit_bp.route('/<ru_ref>/change-respondent-status', methods=['GET'])
+@login_required
+def confirm_change_respondent_status(ru_ref):
+    return render_template('confirm-respondent-status-change.html',
+                           ru_ref=ru_ref,
+                           respondent_id=request.args['party_id'],
+                           first_name=request.args['respondent_first_name'],
+                           last_name=request.args['respondent_last_name'],
+                           email_address=request.args['email_address'],
```

Comment:
We shouldn't be passing email as a query param

Reply:
we shouldn't be...